src/graph-helpers/raw-type-for-field.js

```diff
@@ -0,0 +1,53 @@
+import graphSchema from 'graph/schema';
+
+function findInFields(property, type) {
+  const propertyDescriptor = type.fields[property];
+
+  if (propertyDescriptor) {
+    return {
+      name: 'Literal',
```

Comment:
Shouldn't this be String, or something that describes it's a Scalar of type String?